### ISSUE TITLE
'Add fretboard diagram' option doesn't appear in right click menu during navigation and using Shift+F10 shortcut

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#28646]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

1. Using Alt+Right, navigate to Chord symbol
2. Shift+F10 to trigger context menu
3.  **Add fretboard diagram** option is **missing**

It works with mouse, or select chord with mouse and then trigger Shift+F10

### Supporting files, videos and screenshots

https://github.com/user-attachments/assets/0c21de5d-711f-4391-bbbb-380044cfee31

### What is the latest version of MuseScore Studio where this issue is present?

nightly build 4.6.0-251820328

### Regression

No.

### Operating system

Windows 10

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
